pkgs/tools/security/pcsclite/default.nix

```diff
@@ -25,7 +26,16 @@ stdenv.mkDerivation rec {
     hash = "sha256-nuP5szNTdWIXeJNVmtT3uNXCPr6Cju9TBWwC2xQEnQg=";
   };
 
-  patches = [ ./no-dropdir-literals.patch ];
+  patches = [
+    ./no-dropdir-literals.patch
+    # This patch enables polkit to deny access to specific readers without terminating the entire connection
+    # Estimated to be released in Version >=2.0.1
```

Comment:
That version got already released https://github.com/LudovicRousseau/PCSC/releases/tag/2.0.1